format/spec.md

```diff
@@ -979,18 +980,18 @@ The unified partition type looks like `Struct<field#1, field#2>`.
 
 #### Encryption Keys
 
-Encryption keys and metadata required for decrypting the manifest list files in encrypted tables.
-There are two types of entries:
-1. `key-metadata`: serialized key-metadata of the encrypted manifest list files. The key-metadata objects include encryption keys and other fields required to decrypt a file. Since these objects are sensitive, the serialized key-metadata byte arrays are encrypted by another key. The encryption is done via the integrity-preserving AES GCM cipher, using the snapshot ID as the AAD (additional authentication data) parameter. The result of the encryption is converted to a string via base64 encoding.
-2. `key`: the AES GCM key that encrypts the manifest list key-metadata. Since these keys are sensitive, they are wrapped/encrypted in a Key Management Service (KMS), using the table master key. The result of the wrapping is converted to a string via base64 encoding.
+Keys used for table encryption can be tracked in table metadata as a list named `encryption-keys`. The schema of each key is a struct with the following fields:
 
-`encryption-keys` field of table metadata is an optional list of structs with the following fields:
+| v1 | v2 |     v3     |     Field name          |   Type.              | Description |
+|----|----|------------|-------------------------|----------------------|-------------|
+|    |    | _required_ | **`key-id`**            | `string`             | ID of the encryption key |
+|    |    | _required_ | **`key-metadata`**.     | `string`             | Encrypted key and metadata, base64 encoded [1] |
+|    |    | _optional_ | **`encryption-key-id`** | `string`             | Optional ID of the key used to encrypt `key-metadata` |
+|    |    | _optional_ | **`properties`**        | `map<string, string> | A string to string map of additional metadata used by the table's encryption scheme |
```

Comment:
I'm adding this so that keys can have additional information if needed, like the format used for serialization. We can remove it if you don't think it's necessary but I wanted to make this a bit future-proof.